prbt_hardware_support/test/unittests/unittest_filter_pipeline.cpp

```diff
@@ -35,20 +35,20 @@ class FilterPipelineTest : public testing::Test
  * @brief Tests that exception is thrown if empty callback function is
  * specified.
  */
-//TEST_F(FilterPipelineTest, testEmptyCallbackFunction)
-//{
-//  ros::NodeHandle nh;
-//  FilterPipeline::TCallbackFunc cb;
-//  EXPECT_THROW(FilterPipeline(nh, cb), std::invalid_argument);
-//}
+TEST_F(FilterPipelineTest, testEmptyCallbackFunction)
+{
+  ros::NodeHandle nh;
+  FilterPipeline::TCallbackFunc cb;
+  EXPECT_THROW(FilterPipeline(nh, cb), std::invalid_argument);
+  nh.shutdown();
+}
 
 
 } // namespace filter_pipeline_test
 
 int main(int argc, char** argv)
 {
   ros::init(argc, argv, "unittest_filter_pipeline");
-  ros::NodeHandle nh_;
```

Comment:
This nodehandle is held, to avoid rosconsole::shutdown(), which results in subsequent ROS_* messages no longer output.